app/models/competition.rb

```diff
@@ -1007,6 +1007,10 @@ def registration_currently_open?
     use_wca_registration? && !cancelled? && !registration_not_yet_opened? && !registration_past?
   end
 
+  def after_registration_open?
```

Comment:
The method name doesn't match for what it actually is. For example, if a competition is cancelled after registration is open, this will return `false`, but it is still `after_registration_open`. Same for `use_wca_registration`. How about renaming this method to something like `can_show_competitors_page` and that way you can actually bring the `@competition.any_registrations?` as well to inside this method?

Reply:
(Ended up deciding to keep `after_registration_open?` because otherwise the logic would have just been duplicated in `can_show_competitors_page?` and `registration_currently_open?`